glfw/src/internal_debug.zig

```diff
@@ -0,0 +1,13 @@
+const std = @import("std");
+const zig_builtin = @import("builtin");
+
+// TODO: Consider if this idea could also be applied to prevent 'PlatformError'
```

Comment:
This won't work, GLFW returns platform errors even sometimes in dynamic situations. For example, on Linux if X server does not support specific extensions.

Reply:
Duly noted and done.